src/Language/Fixpoint/Types/Names.hs

```diff
@@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
   , dropSym
   , headSym
   , lengthSym
+  , mappendSym
```

Comment:
Regardless, do not expose mappendSym, you can use symbolSuffix instead...

On Monday, May 16, 2016, Niki Vazou <notifications@github.com> wrote:

> In src/Language/Fixpoint/Types/Names.hs
> <https://github.com/ucsd-progsys/liquid-fixpoint/pull/227#discussion_r63375748>
> :
>
> > @@ -44,6 +44,7 @@ module Language.Fixpoint.Types.Names (
> >    , dropSym
> >    , headSym
> >    , lengthSym
> > +  , mappendSym
>
> Yes, I am creating the symbol from the refined annotation instance
> ToProve a (Proof -> b) where. Let me know if you find a better way:)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ucsd-progsys/liquid-fixpoint/pull/227/files/019c3bdb49e2988b8a59fbff30cd84b7450b862b..1a6a2b4f034faee428e2bf1c6c08dcbfc1238b9c#r63375748>
>